cooltools/api/snipping.py

```diff
@@ -44,6 +44,7 @@
     is_valid_expected,
 )
 from ..lib.common import assign_view_auto, make_cooler_view
+from ..lib.common import pool_decorator
```

Comment:
this could be combined into previous line @Yaoyx

Reply:
combined